library/Freckle/Test/Yesod/RequestBuilder.hs

```diff
@@ -0,0 +1,75 @@
+module Freckle.Test.Yesod.RequestBuilder
+  ( -- * The RequestBuilder type
+    RequestBuilder
+
+    -- * Using a RequestBuilder
+  , request
```

Comment:
Should this be here or in `MakingRequests` though?

Reply:
Maybe exported from both? If I were reading through the `RequestBuilder` module and didn't see any way to execute a `RequestBuilder`, I'd be puzzled. Likewise if I were looking through the `MakingRequests` module to find out what utilities were available for making requests, I'd want to be pointed to the `RequestBuilder` tool.